types/react-native-video/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for react-native-video 1.0
+// Type definitions for react-native-video 2.0
```

Comment:
![image](https://user-images.githubusercontent.com/5743863/29997399-86fe7646-9044-11e7-937b-5e45ffcb4105.png)

Cannot upgrade this version, because the current version of `react-native-video` is 1.0.

Reply:
Yeah, I noticed this, but also saw that the `npm` version is `2.0.0` (https://www.npmjs.com/package/react-native-video). I'm guessing someone released an update to `npm` but not specifically to the github repo.

I can revert this back to 1.0 for the types here though, wasn't sure which one to go with. Thoughts?

Reply:
sorry, I have been negligent.  😳

Reply:
@huhuanming thanks for checking though! 👍